src/components/questionStructure/Task/variants/HandwritingTask/ViewOnlyCanvas.tsx

```diff
@@ -0,0 +1,33 @@
+import { Excalidraw } from '@excalidraw/excalidraw'
+import {
+  ExcalidrawImperativeAPI,
+  ExcalidrawInitialDataState,
+} from '@excalidraw/excalidraw/types/types'
+import { Box, Card } from '@radix-ui/themes'
+import React, { useEffect, useState } from 'react'
+
+export const ViewOnlyCanvas: React.FC<{ initialData: ExcalidrawInitialDataState }> = ({
```

Comment:
Props into interface. Also, could a better name for this be `RawHandwritingPreview`?

Reply:
Moved props into interface in fbbc091. This component is used for both handwriting tasks not just RawHandwriting.

Reply:
Sure ok. I will ponder the naming a little more, but can be refactored after all the PRs have been merged anyways.